pyre/pyre-interpreter/include/pyre3.14t/modsupport.h

```
@@ -0,0 +1,611 @@
+}
+
+static inline int PyArg_ParseTuple(PyObject *args, const char *format, ...)
+{
+    va_list va;
+    va_start(va, format);
+    int parsed = _PyPyre_VaParse(args, NULL, format, NULL, &va, NULL);
+    va_end(va);
+    return parsed;
+}
+
+static inline int PyArg_ParseTupleAndKeywords(PyObject *args, PyObject *kwargs,
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'PyArg_ParseTupleAndKeywords' of similar type ('int *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)